File: Projects/Framework-Samples/NHibernate/VisualBasic/Sample.Data/Base/UnitTestsBase.vb
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports NUnit.Framework
Imports Sample.Data.Generated.ManagerObjects
Imports Sample.Data.Generated.BusinessObjects

Namespace Sample.Data.Generated.Base
	Public Class UNuitTestBase
		Protected managerFactory As IManagerFactory = New ManagerFactory()
	End Class
End Namespace

File: Framework-Samples/NHibernate/CSharp/Sample.Data/Base/UnitTestsBase.cs
using System;
using System.Collections.Generic;
using System.Text;
using NUnit.Framework;
using Sample.Data.Generated.ManagerObjects;
using Sample.Data.Generated.BusinessObjects;

namespace Sample.Data.Generated.Base
{
    public class UnitTestbase
    {
        protected IManagerFactory managerFactory = new ManagerFactory();
    }
}